frontend/src/libs/components/dropdown/dropdown.tsx

```diff
@@ -28,18 +28,20 @@ type Properties<T extends FieldValues> = {
   placeholder?: string;
   field?: ControllerRenderProps<T, FieldPath<T>>;
   className?: string;
+  isCustomValueContainer?: boolean;
 };
 
 const getClassNames = (
   isMenuOpen: boolean,
+  isCustomValueContainer: boolean,
 ): ClassNamesConfig<SelectOption, false, GroupBase<SelectOption>> => ({
```

Comment:
```suggestion
type FooArgs = {
  isMenuOpen: boolean;
  isCustomValueContainer: boolean;
}

const getClassNames = ({
isMenuOpen,
isCustomValueContainer
}: FooArgs): ClassNamesConfig<SelectOption, false, GroupBase<SelectOption>> => ({
```
because many positional arguments (especially booleans) is ambiguous